src/modules/core/components/MembersList/MembersListItem.css

```diff
@@ -26,6 +29,7 @@
 }
 
 .displayName + .username {
```

Comment:
It doesn't really matter because we will always have userName after the display name. And we might not have displayName at all, then we don't care if inline is applied or not. there won't be double stacking of the names